prime-angular-frontend/src/app/core/services/error-handler.service.ts

```diff
@@ -29,6 +32,7 @@ export class ErrorHandlerService implements ErrorHandler {
       }
     } else {
       // Client error has occurred (Angular Error, ReferenceError...)
+      dialogLogger.log(error);
```

Comment:
Running this locally and creating an error it appears to invoke the dialog twice